Provider Provisioner/src/handle-input-offer.ts

```diff
@@ -28,19 +35,49 @@ export async function handleInputOffer(params: HandleInputOfferParams): Promise<
   if (!orders.length) {
     return;
   }
+  await spctlService.initialize();
```

Comment:
It looks weird. The `spctlService` must be initialized before it is passed to the `handleInputOffer` function.

Reply:
We're initializing blockchain here. we dont need to do it before we actually have some orders to complete. Will rename it to initializeBlockchainConnector